iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveShell.java

```diff
@@ -201,6 +201,8 @@ private HiveConf initializeConf() {
 
     // Tez configuration
     hiveConf.setBoolean("tez.local.mode", true);
+    // TODO: enable below option once HIVE-26445 is investigated
+    // hiveConf.setBoolean("tez.local.mode.without.network", true);
```

Comment:
If there are failures due to the change then it becomes a bit doubtful if setting the property always to true is a good idea.

Reply:
Not sure but it says in the description:
```
which is in the same JVM as the client.
```
May be some instance is getting shared by the client and one change is impacting other, hence the NPE which is mentioned in HIVE-26445.

I share the same doubt as @zabetak , we should atleast figure out the reason for failure if it is nothing core-related this property, then this should be ok.

Reply:
recently, only hive iceberg related tests failed [here](http://ci.hive.apache.org/blue/organizations/jenkins/hive-precommit/detail/PR-3491/2/pipeline), and I cannot easily disable this feature only for TestIcebergLlapLocalCliDriver (but not for other CliDrivers), so I'm about to make this change only for some unit tests (which I targeted originally), and handle all the qtests in the followup HIVE-26445, does it make sense?

Reply:
Sounds good.